ironfish-cli/src/utils/ledger.ts

```diff
@@ -17,36 +17,32 @@ import TransportNodeHid from '@ledgerhq/hw-transport-node-hid'
 import { Errors, ux } from '@oclif/core'
 import IronfishApp, {
   IronfishKeys,
+  KeyResponse,
   ResponseAddress,
+  ResponseDkgRound1,
+  ResponseDkgRound2,
+  ResponseIdentity,
   ResponseProofGenKey,
   ResponseSign,
   ResponseViewKey,
 } from '@zondax/ledger-ironfish'
-import {
-  default as IronfishDkgApp,
-  KeyResponse,
-  ResponseAddress as ResponseAddressDkg,
-  ResponseDkgRound1,
-  ResponseDkgRound2,
-  ResponseIdentity,
-  ResponseProofGenKey as ResponseProofGenKeyDkg,
-  ResponseViewKey as ResponseViewKeyDkg,
-} from '@zondax/ledger-ironfish-dkg'
 import { ResponseError } from '@zondax/ledger-js'
```

Comment:
This is a bug because we're importing from a library that we don't have a dependency on. We shouldn't import from dependencies of our dependencies. We need to include this in our package.json.